src/components/dashboard/widget_table/widget_table.gd

```diff
@@ -114,16 +116,27 @@ func set_data(data, type):
 			set_headers(headers)
 	elif data_source_type == DataSource.TYPES.SEARCH:
 		set_headers(DefaultSearchAttributes.keys())
-		update_column_types()
+		
+	update_column_types()
 		
 	update_page_count(raw_data.size())
 	yield(VisualServer, "frame_post_draw")
 	sort_by_column(sorting_column, sorting_type == "asc")
 
 
 func update_column_types():
-	for key in DefaultSearchAttributes.keys():
-		var prop : String = DefaultSearchAttributes[key][-1]
+	var columns_dict := {}
+	if data_source_type == DataSource.TYPES.SEARCH:
+		for key in DefaultSearchAttributes.keys():
+			columns_dict[key] = DefaultSearchAttributes[key][-1]
+	elif "aggregate" in query_structure:
+		for element in query_structure.aggregate.group_by:
+			columns_dict[element["as"]] = element["name"].split(".")[-1]
```

Comment:
Note: this only takes `group_by` into account. There are also `group_func` entries. All entries here would be numbers. Maybe the default sorter already is doing the right thing?

Reply:
Yes, for numeric values the default sort already works.